resource-management/pkg/store/redis/redis.go

```diff
@@ -177,6 +185,49 @@ func (gr *Goredis) PersistVirtualNodesAssignments(virtualNodeAssignment *store.V
 	return true
 }
 
+// Get batch node ids for node status inquiry
+func (gr *Goredis) BatchLogicalNodesInquiry(batchSize int64) []*types.LogicalNode {
```

Comment:
Can batch size be integer only? Since this is an internal function, I highly doubt we need to pull batchSize > max int. Declaring it as anything other than int would cause unnecessary type conversion.

Reply:
Good suggestion. The batchSize has been declared as "int" type.